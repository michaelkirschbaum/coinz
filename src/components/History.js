/** @jsx jsx */
import React, { Component } from 'react'
import { css, jsx } from '@emotion/core'
import * as d3 from 'd3'

class History extends Component {
  constructor(props) {
    super(props)
    this.state = {

    }
  }

  componentDidMount() {
    this.drawChart()
  }

  drawChart() {
    const svg = d3.select('body')
  }

  render() {
    return (
      <div css={css`
        height: 100vh;
        width: 100%;
        border: 1px solid lightgrey;
      `}></div>
    )
  }
}

export default History