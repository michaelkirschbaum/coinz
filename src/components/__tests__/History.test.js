import React from 'react'
import {shallow} from 'enzyme/build';
import History from 'components/History';

describe('History', () => {
  it('should match the snapshot', () => {
    const history = shallow(
      <History />,
    )
    expect(history).toMatchSnapshot()
  })
})