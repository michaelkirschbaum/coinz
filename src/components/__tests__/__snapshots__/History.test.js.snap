// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History should match the snapshot 1`] = `
<EmotionCssPropInternal
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  css={
    Object {
      "map": undefined,
      "name": "7ym0ve",
      "next": undefined,
      "styles": "
        width: 200px;
        height: 200px;
        background: lightblue;
      ",
    }
  }
/>
`;
